src/app/public/modules/inline-delete/inline-delete.component.spec.ts

```diff
@@ -39,23 +40,24 @@ describe('Inline delete component', () => {
     el = fixture.nativeElement;
   });
 
-  it('should emit the deleteTriggered event when the delete button is clicked', () => {
+  it('should emit the deleteTriggered event when the delete button is clicked', fakeAsync(() => {
     fixture.detectChanges();
-    const deleteTriggeredSpy = spyOn(cmp.inlineDelete.deleteTriggered, 'emit').and.callThrough();
-    (<HTMLElement>el.querySelector('.sky-btn-danger')).click();
+    const deleteTriggeredSpy = spyOn(cmp, 'onDeleteTriggered').and.callThrough();
```

Comment:
Testing the `Output` on the consuming component, instead of the inline-delete component, to make sure it makes it "all the way out".